src/Bladeburner/Bladeburner.ts

```diff
@@ -750,31 +751,26 @@ export class Bladeburner {
     }
   }
 
+  private getActionCasualties(action: Operation | BlackOperation, success: boolean) {
+    const severity = success ? CasualtyFactor.LOW_CASUALTIES : CasualtyFactor.HIGH_CASUALTIES;
+    const casualties = new TeamCasualties(severity, action.teamCount, this.sleeveSize);
+    return casualties.rollOutcome(getRandomIntInclusive, this);
+  }
+
+  public killSupportingSleeves(sleeveDeaths: number) {
+    const sup = Player.sleeves.filter((x) => isSleeveSupportWork(x.currentWork));
+    const damagedIndices = [...Array(Math.min(sup.length, sleeveDeaths)).keys()].sort(() => 0.5 - Math.random());
```

Comment:
This is just a nitpick.

If you want to shuffle an array, you should use Fisher–Yates shuffle, not this `0.5 - Math.random()` trick. We have duplicated implementation of Fisher–Yates shuffle in our code, and we even import `shuffle` from `lodash`. If you want to use Fisher–Yates shuffle, you can use one of them. I'll clean them up later. If you think that it's unnecessary, you can ignore this comment. It's not a big deal in our use case. There are other problems with our randomization functions that I turn a blind eye to.

Reply:
Can you show me a sample usage of our internal fisher-yates?

Reply:
You can search the usages of `shuffleArray` in `src\Infiltration\ui\BribeGame.tsx` and `shuffle` in `src\data\codingcontracttypes.ts`.

Reply:
Will do, thanks!